python/sglang/srt/models/deepseek_nextn.py

```diff
@@ -94,7 +94,9 @@ def forward(
         zero_allocator = BumpAllocator(
             buffer_size=2,
             dtype=torch.float32,
-            device=input_ids.device,
+            device=(
+                input_embeds.device if input_embeds is not None else input_ids.device
```

Comment:
Why do we need to update deepseek_nextn?

Reply:
Because it has input_embeds field and related logics (if input_embeds is None: ... else: ...), so I do it to be safe.